test/main.js

```diff
@@ -17,12 +17,15 @@ import edge from 'selenium-webdriver/edge'
 let config
 const browsersFailures = {}
 let totalFailures = 0
+export let localhostUrl
 
 if (!process.env.CONFIG_FILE) {
   console.error('INFO: CONFIG_FILE not set, so using the default config.json')
   config = require('./config')
+  localhostUrl = config.localhostUrl
 } else {
   config = require(`./${process.env.CONFIG_FILE}`)
+  localhostUrl = config.localhostUrl
```

Comment:
And here I would do this: 
```suggestion
  const envConfig = require(`./${process.env.CONFIG_FILE}`)
  localhostUrl = envConfig.localhostUrl
```

Reply:
Noted, I'll make that change.